src/App/mermaidData/databaseSwitchboard/ManagementRegimesMixin.js

```diff
@@ -39,24 +40,38 @@ const ManagementRegimesMixin = (Base) =>
         : Promise.reject(this._notAuthenticatedAndReadyError)
     }
 
+    getManagementRegimesFromApi = async function getManagementRegimesFromApi(projectId, pageNo) {
+      if (!projectId) {
+        return Promise.reject(this._operationMissingParameterError)
+      }
+
+      return this._isOnlineAuthenticatedAndReady
+        ? axios
+            .get(`${this._apiBaseUrl}/managements/`, {
+              params: {
+                exclude_projects: projectId,
+                include_fields: `project_name`,
+                page: pageNo,
+                limit: DEFAULT_QUERY_LIMIT,
+              },
+              ...(await getAuthorizationHeaders(this._getAccessToken)),
+            })
+            .then((apiResults) => apiResults.data)
+        : Promise.reject(this._notAuthenticatedAndReadyError)
+    }
+
     getManagementRegimesExcludedInCurrentProject =
-      async function getManagementRegimesExcludedInCurrentProject(projectId) {
-        if (!projectId) {
-          return Promise.reject(this._operationMissingParameterError)
+      async function getManagementRegimesExcludedInCurrentProject(projectId, pageNo = 1) {
+        const apiResultData = await this.getManagementRegimesFromApi(projectId, pageNo)
+        const { results, count } = apiResultData
+
+        if (pageNo * DEFAULT_QUERY_LIMIT < count) {
```

Comment:
Im confused about how pageNo, DEFAULT_QUERY_LIMIT and count relate to eachother. 

What can we call the condition `pageNo * DEFAULT_QUERY_LIMIT < count` to describe that state where this is true?